test/e2e/autoscaling/cluster_autoscaler_scalability.go

```diff
@@ -209,7 +216,9 @@ var _ = framework.KubeDescribe("Cluster size autoscaler scalability [Slow]", fun
 		rcConfig := reserveMemoryRCConfig(f, "some-pod", replicas, replicas*perNodeReservation, largeScaleUpTimeout)
 		expectedResult := createClusterPredicates(totalNodes)
 		config := createScaleUpTestConfig(totalNodes, totalNodes, rcConfig, expectedResult)
-		testCleanup := simpleScaleUpTestWithEpsilon(f, config, 0.1)
+		tolerateUnreadyNodes := totalNodes / 10
+		tolerateUnreadyPods := replicas / 10
```

Comment:
why 10?

Reply:
It used to be 10% (epsilon = 0.1), so I translated this from relative to absolute. I have no clue where the 0.1 number comes from.